libraries/app/database_api.cpp

```diff
@@ -87,12 +87,15 @@ class database_api_impl : public std::enable_shared_from_this<database_api_impl>
 
       // Accounts
       account_id_type get_account_id_from_string(const std::string& name_or_id)const;
-      vector<optional<account_object>> get_accounts(const vector<std::string>& account_names_or_ids)const;
-      std::map<string,full_account> get_full_accounts( const vector<string>& names_or_ids, bool subscribe );
+      vector<optional<account_object>> get_accounts( const vector<std::string>& account_names_or_ids,
+                                                     optional<bool> subscribe )const;
+      std::map<string,full_account> get_full_accounts( const vector<string>& names_or_ids, optional<bool> subscribe );
```

Comment:
Please wrap.

Reply:
I thought 118 characters was fine..
This looks good in Firefox, but I'd admit it's not very nice in Chrome.